src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -1682,7 +1682,7 @@ private void emitNoResolvedType(NoType type, String defaultEmit) {
       if (importRenameMap.containsKey(displayName)) {
         displayName = importRenameMap.get(displayName);
       }
-      emit(Constants.INTERNAL_NAMESPACE + "." + displayName);
+      emit(displayName);
```

Comment:
Leave some trace in comment why we switched to 'display name' vs 'Constants.INTERNAL_NAMESPACE + "." + displayName.'